server/src/main/java/thesistrack/ls1/entity/ThesisAssessment.java

```diff
@@ -0,0 +1,53 @@
+package thesistrack.ls1.entity;
+
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.Instant;
+import java.util.UUID;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "thesis_assessments")
+public class ThesisAssessment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "assessment_id", nullable = false)
+    private UUID id;
+
+    @NotNull
+    @OneToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "thesis_id", nullable = false)
+    private Thesis thesis;
+
+    @NotNull
+    @Column(name = "summary", nullable = false, length = Integer.MAX_VALUE)
```

Comment:
MAX_VALUE values are huge objects in the DB. Assuming 3000 chars/page, I do not think that a summary will even be remotely close to 715.000 pages 😮 Please reconsider your `length` parameter

Reply:
I did not really want that admins run into limits, but yeah makes sense to change it to something more reasonable